international-telephone-input/functions/lookup.js

```diff
@@ -0,0 +1,55 @@
+/**
+ *  Lookup - validate a phone number
+ *
+ *  This function will tell you whether or not a phone number is valid using Twilio's Lookup API
+ *
+ *  Parameters:
+ *  "phone" - string - phone number in E.164 format (https://www.twilio.com/docs/glossary/what-e164)
+ *
+ *  Returns JSON
+ *  {
+ *    "success": boolean,
+ *    "error": string      // not present if success is true
+ *  }
+ */
+
+exports.handler = function (context, event, callback) {
+  const response = new Twilio.Response();
+  response.appendHeader("Content-Type", "application/json");
+
+  // uncomment to support CORS
+  // response.appendHeader('Access-Control-Allow-Origin', '*');
+  // response.appendHeader('Access-Control-Allow-Methods', 'POST, OPTIONS');
+  // response.appendHeader('Access-Control-Allow-Headers', 'Content-Type');
+
+  if (typeof event.phone === "undefined") {
```

Comment:
In my testing this could never evaluate to `true` because the front-end JS sets the phone parameter to an empty string. Oddly, this also results in a positive lookup from the API and a success message that shows no phone number.

We could change it to:

```js
if (event.phone === '') {
```

and the error will be returned in the following block.